webapp/templates/proxy-create.xhtml

```diff
@@ -0,0 +1,150 @@
+        #ruleTable td:nth-child(2) { 
+            border-right:1px solid black;
+        }
+        .selectize-input {
+            margin-top:4px;
+        }
+    </style>
+    <script type="text/javascript" src="../scripts/purl.js"></script>
+</head>
+<body>
+    <form role="form" id="form" method="POST" action="" enctype="text/turtle" typeof="calli:Proxy" class="container">
+        <div class="page-header">
+            <h1>New Proxy</h1>
+        </div>
+        <div class="row">
+            <div class="col-sm-6">
+                <fieldset>
+                    <div class="form-group">
+                        <label for="local">Local name</label>
+                        <div>
+                            <input type="text" id="local" value="{rdfs:label}" class="form-control" required="required" autofocus="autofocus"
+                                onchange="calli.updateProperty(event, 'rdfs:label')" />
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="comment">Comment</label>
+                        <div>
+                            <textarea id="comment" class="form-control"
+                                onchange="calli.updateProperty(event, 'rdfs:comment')">{rdfs:comment}</textarea>
+                        </div>
+                    </div>
+                </fieldset>
+            </div>
+            <div class="col-sm-6">
+                <aside class="well">
+                    <p>PURLs are used to curate the URL resolution process, thus solving
+                    the problem of transitory URIs in location-based URI schemes like HTTP.</p>
+                    <p>Partial PURLs are a special class of PURL that must end with a '/' and 
+                    allow a hierarchy of URLs to be mapped to another resource. The request pattern
+                    may include regular expressions in the path subsequent to the PURL name.</p>
+                    <p>Other URLs cannot end with a '/' and the request pattern only applies to the query string.</p>
+                </aside>
+            </div>
+        </div>        
+                
+        <div id="ruleOptions" style="display:none">
+            <!-- These are not used by the template but are required for validation when the resource is created -->
+            <input type="hidden" property="calli:copy" content="?copy" value="{?copy}"/>
+            <input type="hidden" property="calli:post" content="?post" value="{?post}"/>
+            <input type="hidden" property="calli:put" content="?put" value="{?put}"/>
+            <input type="hidden" property="calli:patch" content="?patch" value="{?patch}"/>
+            <input type="hidden" property="calli:delete" content="?delete" value="{?delete}"/>            
+        </div>
+                
+        <div class="row">
+            <div class="col-sm-12">
+                <label>Rules
```

Comment:
I think this Proxy UI could be simpler. I think the UI from Callimachus 1.4 Purl create form would be a better starting point (then copying RewriteRule create form): a section for each client request method. This way the LHS is already taken care for them and they just need to fill in the RHS with what action the system should take for each request method. The regular expression is there only to allow the user to extract variables (like path segments) for use in the template and should be part of the RHS of the rule, since the request method must already be unique.